docker/alarms.toml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Abstract Machines
+# SPDX-License-Identifier: Apache-2.0
+
+# To listen all messsage broker subjects use default value "alarms.>".
+# To subscribe to specific subjects use values starting by "alarms." and
+# followed by a subtopic (e.g ["alarms.<domain_id>.c.<channel_id>.sub.topic.x", ...]).
+[subscriber]
+subjects = ["alarms.>"]
+
+[transformer]
+format = "gob"
```

Comment:
We can avoid this config, if we directly use `Subscriber`.  
Because consumer will have nil transformer.

Reply:
I previously suggested the idea of removing the consumer and using the subscriber directly to @dborovcanin, and I believe he is aligned with the switch.